CHANGELOG.md

```diff
@@ -26,6 +26,60 @@ Packages are formatted by release version. You can see all the changes done to p
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html) as explained in the [Backward Compatibility Promise](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/).
 
+## [v11.1.0](https://github.com/shopsys/shopsys/compare/v11.0.0...v11.1.0) (2023-07-18)
+
+### :sparkles: Enhancements and features
+* [shopsys] added detailed cron run information for administrators by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2581
+* [shopsys] adverts on product list can now be set only for range of categories by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2583
+* [shopsys] added ability to change periodicity and length of cron runs by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2584
+* [framework] columns in grid can now be reordered by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2564
+* [shopsys] added ability to change content of robots.txt file through administration by @radek-bruha in https://github.com/shopsys/shopsys/pull/2591
+* [shopsys] users and admins now have only five tries to log in one minute by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2599
+* [framework] cron overview is now available for admin in the default state by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2600
+* [shopsys] added ability to change email whitelist through administration by @radek-bruha in https://github.com/shopsys/shopsys/pull/2592
+* [shopsys] don't show 500 error page when you have 405 http code by @sspooky13 in https://github.com/shopsys/shopsys/pull/2666
+
+### :bug: Bug Fixes
+* [project-base] returned symfony.lock to project-base as it caused problems with symfony flex installing packages again by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2595
```

Comment:
I would not mention this as it matters only for releaser

Reply:
removed in c110a6e27